app/fastcgi_cert_manager.c

```diff
@@ -0,0 +1,374 @@
+#include "fastcgi_cert_manager.h"
+#include <fcgi_config.h>
+#include <glib-object.h>
+#include <glib.h>
+#include <glib/gstdio.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <sys/stat.h>
+#include <syslog.h>
+#include <unistd.h>
+#include "fcgi_stdio.h"
+#include "uriparser/Uri.h"
+
+#define FCGI_SOCKET_NAME "FCGI_SOCKET_NAME"
+
+#define syslog_v(...) if (g_verbose) {syslog(__VA_ARGS__);}
+
+
+static const char *app_localdata_path =
```

Comment:
could we save to /tmp here in the cert manager instead?

Reply:
Think it should be possible?.. but would mean a copy and delete instead of the current rename of the temp file..

Reply:
An implementation of this pushed for re-review